queries/dart/highlights.scm

```diff
@@ -83,22 +94,49 @@
 
 ; Variables
 
+; var keyword
+(inferred_type) @keyword
```

Comment:
Is this really a type ? If it is we should use `@type` here.

Reply:
It might just be the naming but it isn't a type. In dart using the `var` keyword will infer the type of a variable. Which I think is why this is called inferred type in the grammar but it actually refers to the `var` keyword.